### PR TITLE
fix: conditional statements jinja2 warning

### PR DESCRIPTION
**Pull request by iisojunn**
When ansible-core version is updated from 2.15.4 to 2.15.8 the following warning occurs:
[WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}

Fixed by removing the delimeters.

### Proposed changes

Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue using one of the [supported keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) here in this description (not in the title of the PR).

### Checklist

Before creating a PR, run through this checklist and mark each as complete:

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginxinc/ansible-role-nginx/blob/main/CONTRIBUTING.md) document.
- [x] I have added Molecule tests that prove my fix is effective or that my feature works.
- [x] I have checked that any relevant Molecule tests pass after adding my changes.
- [x] I have updated any relevant documentation ([`defaults/main/*.yml`](https://github.com/nginxinc/ansible-role-nginx/blob/main/defaults/main/), [`README.md`](https://github.com/nginxinc/ansible-role-nginx/blob/main/README.md) and [`CHANGELOG.md`](https://github.com/nginxinc/ansible-role-nginx/blob/main/CHANGELOG.md)).

### CODE REVIEW

**Review by alessfg — APPROVED** [review]

File: tasks/validate/validate.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 - name: Check whether you are using a supported NGINX distribution
   ansible.builtin.assert:
     that:
-      - "{{ ansible_facts['distribution'] | lower in nginx_distributions.keys() | list }}"
-      - "{{ (ansible_facts['distribution_version'] | regex_search('\\d{1,2}\\.\\d{2}') | float in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | map('float'))
-        if ansible_facts['distribution'] | lower in ['alpine', 'ubuntu'] else ansible_facts['distribution_major_version'] in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | string }}"
-      - "{{ ansible_facts['architecture'] in nginx_distributions[ansible_facts['distribution'] | lower]['architectures'] }}"
+      - ansible_facts['distribution'] | lower in nginx_distributions.keys() | list
+      - (ansible_facts['distribution_version'] | regex_search('\\d{1,2}\\.\\d{2}') | float in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | map('float')
```
Comment (alessfg) [inline tasks/validate/validate.yml:6]: Can you move the parentheses back to where they were? I've run into situations using the "X if Y" conditional in Ansible has led to issues when the "X" was not wrapped in parenthese :)

**Review by alessfg — CHANGES_REQUESTED** [review]
See the comment I made in the review 😄

**Comment by iisojunn** [comment]
Based on my manual testing, the short parenthesis (code before if) did not
add any value and therefore removed them.

The two line block did not work without parenthesis around the whole block.
Therefore added them.

Feel free to add the short parenthesis back. I can do it next Tuesday.


pe 12. tammik. 2024 klo 22.53 Alessandro Fael Garcia <
***@***.***> kirjoitti:

**Comment by alessfg** [comment]
It's not needed in this case, but it's there for consistency around this role and the config role. That being said, after removing the curly braces and double quotes, something (e.g. parentheses) had to be used to ensure that it works given that it's a multiline conditional. I'll readd them on Monday 😄

**Review by alessfg — APPROVED** [review]

**Comment by alessfg** [comment]
I ended up flattening everything into a single line 👌

**Pull request merged**